ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -230,18 +273,45 @@ if [[ "${APPLY_ALL_SUBNETS}" == "no" ]]; then
         compute_subnets_len=$(cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" '[.subnets | .[] | select(.zone.name==$z) | select(.name|test("compute"))] | length')
         compute_idx=$(( RANDOM % compute_subnets_len))
       ;;
+      "2Paged")
+        cp_idx="0"
+        compute_idx="0"
+      ;;
       *)
-        echo "unsupported value for PICKUP_SUBNETS_IN_ORDER"
+        echo "unsupported value for PICKUP_SUBNETS_ORDER"
         exit 2
       esac
-      cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" --argjson idx "$cp_idx" '[[.subnets | sort_by(.created_at) | .[] | select(.zone.name==$z) | select(.name|test("control-plane")) | .name][$idx]]' | yq-go r -P - >>${workdir}/controlPlaneSubnets.yaml
-      cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" --argjson idx "$compute_idx" '[[.subnets | sort_by(.created_at) | .[] | select(.zone.name==$z) | select(.name|test("compute")) | .name][$idx]]' | yq-go r -P - >>${workdir}/computerSubnets.yaml
+      
+      if [[ $PICKUP_SUBNETS_ORDER == "2Paged" && ( ! ${cpHas2Paged} || ! ${computerHas2Paged} ) ]]; then
```

Comment:
The new change makes the logic be worse for reading. Why not check if the picked-up subnets is on the 2nd page, but not changed the original logic. I also proposed a basic logic mentioned in https://github.com/openshift/release/pull/58471#discussion_r1835898279

Reply:
then will based on ibmcloud return the subnets with sort_by(.created_at)?